pyproject.toml

```diff
@@ -49,8 +49,10 @@ requests = "^2.21"
 email_validator = "^1.0"
 pydantic = "^1.4"
 typing_extensions = [
-    { version = "^3.7.4", python = "^3.6" },
-    { version = "^4", python = "^3.7" },
+    { version = "^3.7.4", python = "~3.6" },
+    { version = "^4", python = "~3.7" },
+    { version = "^4", python = "~3.8" },
+    { version = "^4", python = "~3.9" },
```

Comment:
This could be replace by `{ version = "^4", python = ">=3.7" }`. See doc: https://python-poetry.org/docs/dependency-specification/#multiple-constraints-dependencies

Reply:
Also, I cannot see the unit tests on your MR ? I think you need to enable the CI on your forked project for it to be visible in the Pull request.